Jenkinsfile

```
@@ -44,7 +44,8 @@ pipeline {
             cambpmRunMaven('.',
```

We can add a code comment above the cambpmRunMaven command to explain why we're using JDK 11 here, to prevent any (accidental) changes in the future. 

```suggestion
            // we use JDK 11 to build the artifacts, as it is required by the Quarkus extension
            cambpmRunMaven('.',
```

---

👍